README.md

```diff
@@ -28,17 +28,27 @@ Backend: Node.js, Express.js, PostgreSQL (Railway)
 
 Fridge is the unit of food inventory. Users can create and delete fridges. Within each fridge, users can manage food items by adding, updating, and removing. Additionally, users can manage members to share the info with by adding and removing. 
 
+- Managers can create and delete fridges.
+- Fridge should have a name and creator.
+
```

Comment:
I added a "manager" role, because I'm thinking that probably it's better to have someone control the fridge, instead of giving every member the autority to make big changes to a fridge.